root/programs/C#/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx

```diff
@@ -117,15 +117,18 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="gbxDataProviders.Text" xml:space="preserve">
-    <value>データプロバイダを選択する</value>
-  </data>
   <assembly alias="System.Drawing" name="System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
   <data name="Close_ToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
     <value>58, 20</value>
   </data>
-  <data name="Close_ToolStripMenuItem.Text" xml:space="preserve">
-    <value>閉じる </value>
+  <data name="cbxOnlyTableMaintenance.Size" type="System.Drawing.Size, System.Drawing">
+    <value>383, 17</value>
+  </data>
+  <data name="$this.Text" xml:space="preserve">
+    <value>棟梁：D層自動生成ツール（墨壺） - D層、Dao・SQLファイル生成 </value>
+  </data>
+  <data name="btnDaoAndSqlGen.Text" xml:space="preserve">
+    <value>－ Dao・SQL、DTOファイルを生成する －1</value>
```

Comment:
@daisukenishino 

>Maybe, I think that it changed as follows sometime.

I think that what you should do is only deleting "1" of the end.
So, I think that you can deal with the above in this pull request.

>```TXT
>－ Dao・SQL、DTOファイルを生成する －
>```
>▼ ▼ ▼
>```TXT
>－ Dao・SQL、DTOファイルを生成する －1
>```

I think that the direction is the wrong way around.
```TXT
－ Dao・SQL、DTOファイルを生成する －1
```
▼ ▼ ▼
```TXT
－ Dao・SQL、DTOファイルを生成する －
```

>English translation also have a problem

What do you think the **problem**?